.github/workflows/TagBot.yml

```diff
@@ -1,15 +1,15 @@
 name: TagBot
 on:
-  schedule:
-    - cron: 0 0 * * *
-  push:
-    branches:
-      - actions/trigger/TagBot
+  issue_comment:
+    types:
+      - created
```

Comment:
```suggestion
  issue_comment:
    types:
      - created
  push:
    branches:
      - actions/trigger/TagBot
```

Reply:
Out of curiousity, what are you using this for?